### DISCUSSION TITLE
Open external audio

### DISCUSSION THREAD

**Discussion by minh3339** [moneytoo/Player#226] [Ideas]
I hope the developer can add features to select external audio (like mpv-android) while playing the video.
Like this:
![Screenshot_20211210-135657](https://user-images.githubusercontent.com/79286268/145531896-9f81cc06-eeb1-4fee-a02e-73bf313a6623.jpg)

**Comment by BugmanSpirit** [comment]
There is a yt-dlp wrapper app for android, [DVD](https://github.com/yausername/dvd). YouTube video links nowadays come with separate streams for audio and video. You can open the videos in player using share link but there is no way to add audio using external audio, unlike mpv.

A simple link icon next to the file picker icon would be a good addition. With the ability to select links for audio, video and subtitles.
Another option would be to add the external audio option in the audio track menu.
